docs/FIRRTLAnnotations.md

```diff
@@ -0,0 +1,717 @@
+# FIRRTL Annotations
+
+The SFC provides a mechanism to encode arbitrary metadata and associate it with
```

Comment:
Super nit, but I'd say "Scala FIRRTL Compiler (SFC)" to make the doc more self contained.

Reply:
Agree. A reasonable policy is to define everything the first time in every separate doc.